app/templates/ember-community-survey-2020.hbs

```diff
@@ -5,114 +5,375 @@
+        </blockquote>
+      </EsCard>
 
-        <p>
-          Submissions close on March 8th. To ensure the survey captures a fair measure of our community, please share this page on your social networks and with your local communities!
+      <EsCard class="my-2">
+        <p class="text-center large">
+          Score: 3
         </p>
-      </div>
 
-      <div class="lg:col-2">
-        <img
-          alt="Emberconf during talk"
-          class="survey-image"
-          src="/images/survey/photo3.jpg"
-        >
+        <blockquote class="quote">
+          (translated) The distribution of this framework in China is not very high. In addition to the absence of relevant Chinese documents, the number of users is very small.
+        </blockquote>
+      </EsCard>
 
-        <h3>
-          Results at EmberConf
-        </h3>
+      <EsCard class="my-2">
+        <p class="text-center large">
+          Score: 5
+        </p>
 
-        <p>
-          At EmberConf, the results to the 2020 survey will be shared with a comparison to the <LinkTo @route="ember-community-survey-2019">2019 results</LinkTo>. Detailed aggregate and individual responses will also be made public.
+        <blockquote class="quote">
+          Ember is the anti-fatigue framework. It doesn’t pretend to be all things to all people. It acknowledges its flaws while also making clear the plan to address them. It details a clear roadmap for the future and delivers on it incrementally.  Simply put, working with Ember makes me happy.
+        </blockquote>
+      </EsCard>
+
+      <EsCard class="my-2">
+        <p class="text-center large">
+          Score: 5
         </p>
-      </div>
-    </div>
 
-    <p class="mt-4 text-sm">
-      Photos by <a href="http://www.octolabs.com/" rel="noopener noreferrer" target="_blank">Jeremy Green</a>
+        <blockquote class="quote">
+          Ember is powerful and flexible, without being overly complicated. There's a wealth of learning materials, and strong community support, plus the ease of updating to gain the latest features and fixes "for free" is a big bonus.
+        </blockquote>
+      </EsCard>
+    </ul>
+  </:body>
+</EmberCommunitySurvey::Section>
+
+<EmberCommunitySurvey::Section
+  @hasDarkBackground={{true}}
+  @sectionId="ember-version"
+  @sectionTitle="Ember Version"
+>
+  <:charts>
+    <Highcharts
+      @chartOptions={{this.MS_Q401.options}}
+      @data={{this.MS_Q401.data}}
+    />
+  </:charts>
+
+  <:body>
+    <p>
+      At the time the survey was released, Ember 3.16 became the latest stable release of the framework.
+      Legacy 2.x and 1.x numbers dropped, indicating a continued upgrade process for developers on legacy versions of Ember.
+    </p>
+  </:body>
+</EmberCommunitySurvey::Section>
```

Comment:
I was curious. How did you find working with the new `<EmberCommunitySurvey::Section>` component?

I used named blocks to help a new contributor figure out where the charts and body go. Inside the `<:body>` block, the contributor, ideally, only needs to use basic HTML tags like `<p>` and `<ul>` to summarize a survey question, without having to know how to create a section that is semantic and what CSS classes to use to style the section.

I didn't like having to alternate between `@hasDarkBackground={{true}}` and `@hasDarkBackground={{false}}`, but haven't found yet a simpler solution (that accounts for the last section, which has the same background color as the penultimate one...).

Reply:
It was intuitive enough; I could see the pattern and imitate it- I spent more time in the controller file working with the data (which still needs some cleanup) 👍